docs/en/api/POSIX-API.md

```diff
@@ -169,6 +171,26 @@ pid mount_point alluxio_path
 
 ## Advanced Setup
 
+### Select which libfuse version to use
+
+Alluxio now supports both libfuse2 and libfuse3. libfuse2 is more stable and more tested in production. libfuse3 support is now experimental. However, libfuse3 is newer and actively developed, and alluxio is working on utilizing the new features provided by libfuse3.
```

Comment:
is more stable and has been tested in production. Libfuse3 is experimental but under active development. Alluxio will focus more on libfuse3 and utilize new features provided.

Reply:
libfuse 2 and 3 are actually both stable and production ready. It is the alluxio's support on libfuse3 that is experimental. I think it is more precise to say "Alluxio FUSE on libfuse2 is more stable and has been tested in production. Alluxio FUSE on libfuse3 is currently experimental but under active development. Alluxio will focus more on libfuse3 and utilize new features provided.".